.vscode/launch.json

```diff
@@ -0,0 +1,47 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
```

Comment:
This is not standard json, but created by the extension. Should we keep this?
+: gives info on how to use it
-: standard parser (if we ever wanted to use one, although I don't know why) could complain/break (like the github parser here)